worker/jobTypes/githubJob.js

```diff
@@ -40,6 +42,56 @@ class GitHubJobClass {
         }
         return false;
     }
+    
+    async writeEnvFile(isProdDeployJob){
+      let pathPrefix;
+
+      if(isProdDeployJob){
+        //download published branches file to check if repo is versioned 
+        const repoObject = {
+          repoOwner: this.currentJob.payload.repoOwner, repoName: this.currentJob.payload.repoName,
+        };
+        const repoContent = await workerUtils.getRepoPublishedBranches(repoObject)
+        //versioned repo
+        if(repoContent && repoContent.content.version.active.length > 1){
+          pathPrefix = `${this.currentJob.payload.repoName.replace('docs-','')}/docsworker-xlarge/${this.currentJob.payload.branchName}`; 
```

Comment:
I'm not sure that our already-published prefixes always match the reponame. Is this verified? For example, cloud-docs, mms-docs, etc.

Also, we don't typically deploy with a user in the prefix. Only stage.